modules/mobile/app/models/mobile/user.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Mobile
+  class User < ApplicationRecord
+    validates :icn, presence: true, uniqueness: true
+
+    after_initialize :set_defaults
+
+    def increment_vet360_link_attempts
+      self.vet360_link_attempts += 1
+    end
+
+    def self.can_count_attempts?
```

Comment:
Did we confirm that this is what they meant by saying the code must be backwardly compatible? Because this feels like a fragile anti-pattern.